bosh/opsfiles/development.yml

```diff
@@ -0,0 +1,524 @@
+# This file is meant to edit the pipeline.yml to make it suitable for the dev environment
+# From looking at the pipeline.yml, it seems like we ought to be able to just update the plan
+# sections, because the others use the anchor, but the anchor is resolved before the
+# opsfile gets at the YAML (https://github.com/cloudfoundry/bosh-cli/issues/254), so 
+# we have to edit all sections.  Sigh.
+
+# these are taken care of in prod.
+- type: remove
+  path: /jobs/name=plan-dns?
+
+- type: remove
+  path: /jobs/name=apply-dns?
+
+## plan-bootstrap-tooling
```

Comment:
Yeah, that whole approach got a bit... larger than expected, I think.  I did it this way because Will said that this was a good approach.  If we have one pipeline that we alter according to the environment it is being deployed into, that's not unreasonable.  I think that we both assumed that the number of changes would be small, and thus this would end up being a much smaller change to the overall project than me refactoring all this stuff to be variables.  It did keep the changes to the project in other areas fairly minimal, but it turns out that the opsfile ended up getting big and hairy.

I'm not sure where to go with this.  I think that it's not a bad approach.  I like having only one pipeline to maintain.  But we certainly could shrink this opsfile a lot if we moved a lot of these things into variables.  Do you see that as a clear win over just keeping these changes encapsulated in this opsfile?